react-toggle-switch-component/src/toggle-switch.jsx

```diff
@@ -5,25 +5,20 @@ export function ToggleSwitch() {
   function handleClick() {
     setIsOn(!isOn);
   }
-  if (isOn) {
-    return (
-      <div className='box'>
-        <label className="switch">
-          <input onClick={handleClick} type="checkbox"></input>
-          <span className="slider round"></span>
-        </label>
-        <h4>ON</h4>
-      </div>
-    );
+
+  function setText() {
+    if (isOn) {
+      return 'ON';
+    }
+    return 'OFF';
   }
   return (
     <div className='box'>
       <label className="switch">
         <input onClick={handleClick} type="checkbox"></input>
         <span className="slider round"></span>
       </label>
-      <h4>OFF</h4>
+      <h4>{setText()}</h4>
```

Comment:
or more simply, `<h4>{isOn ? 'ON' : 'OFF'}</h4>`

Once you get used to it, the ternary operator provides a simple, succinct way to make assignments. But don't use it for complex code. It's best if all ternary operators stay on one line.